skabase/SKALogger/SKALogger.py

```diff
@@ -258,12 +265,15 @@ def SetStorageLoggingLevel(self, argin):
         storage_logging_device = argin[1][:]
         i = 0
         while i < len(storage_logging_level[:]):
-            self.info_stream("Storage logging level : %s, Device : %s",
-                             storage_logging_level[i],
-                             storage_logging_device[i])
-            dev_proxy = DeviceProxy(storage_logging_device[i])
-            dev_proxy.storageLoggingLevel = storage_logging_level[i]
-            i += 1
+            try:
+                self.info_stream("Storage logging level : %s, Device : %s",
+                                 storage_logging_level[i],
+                                 storage_logging_device[i])
+                dev_proxy = DeviceProxy(storage_logging_device[i])
+                dev_proxy.storageLoggingLevel = storage_logging_level[i]
+                i += 1
```

Comment:
Same comments as above.

Reply:
Added original exception message. Commit id 35e9e42 addresses this.